.speakeasy/gen.yaml

```diff
@@ -13,13 +13,18 @@ generation:
     oAuth2ClientCredentialsEnabled: true
     oAuth2PasswordEnabled: true
 typescript:
-  version: 0.4.0
+  version: 0.5.0
   additionalDependencies:
     dependencies:
       jwt-decode: ^4.0.0
+      yaml: ^2.7.0
     devDependencies:
       '@types/deno': ^2.0.0
+      '@types/jest': ^29.5.14
       '@types/node': ^18.0.0
+      jest: ^29.7.0
+      ts-jest: ^29.2.5
```

Comment:
I'm a bit confused, yesterday we discussed that Speakeasy is also using vitest and not jest. Why are we doing jest instead?

Reply:
I had already went deep on jest so stuck with it, also vitest has no real advantage for our use case. We just the basic for which jest works.